clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -224,7 +228,9 @@ PointerVariableConstraint::PointerVariableConstraint(const QualType &QT,
     std::string TyName = tyToStr(Ty);
     if (isVarArgType(TyName)) {
       // Variable number of arguments. Make it WILD.
-      vars.push_back(CS.getWild());
+      VarAtom *WildVA = CS.getFreshVar(Npre + N, VK);
+      CS.createGeq(WildVA, CS.getWild(), "Variable number of arguments.");
```

Comment:
The above two lines are the same two that are at the start of `PointerVariableConstraint::getWildPVConstraint`. Do they occur anywhere else? If this is a standard pattern for introducing a WILD variable it would be nice to abstract it.